index.html

```diff
@@ -1916,9 +1906,9 @@ <h3>Verification Material</h3>
     }
   }, {
     "id": "did:example:123456789abcdefghi#keys-1",
-    "type": "Ed25519VerificationKey2018", <span class="comment">// external (property value)</span>
+    "type": "Ed25519VerificationKey2020", <span class="comment">// external (property value)</span>
```

Comment:
What does "external" mean here? I assume it means that the property value `Ed25519VerificationKey2020` is defined outside of DID Core, e.g. in the CCG's [Security Vocabulary](https://github.com/w3c-ccg/security-vocab), or somewhere in the new LDS WG, or in a cryptographic suite. This is good!

Reply:
Yes, your assumption is correct. Someone (can't remember now) asked for all "external properties" to be clearly marked in the specification and @rhiaro went through and did that a while ago.